implementations/rust/ockam/ockam_entity/src/credential/traits.rs

```diff
@@ -24,7 +35,7 @@ pub trait CredentialIssuer {
         &mut self,
         schema: &CredentialSchema,
         attributes: &[CredentialAttribute],
-    ) -> Result<Credential>;
+    ) -> Result<BbsCredential>;
```

Comment:
This seems like a big choice since it is exposed to the public API. We likely won't always be tied to BBS+, do we want to expose that detail?

Reply:
Since we now use Credential for a type that is used in user code, we need another name for this type. Suggestions?

Reply:
Let's rename later, good for now